packages/core/src/vuePlugins/actions/useExtendedMasterportalapiMarkers.ts

```diff
@@ -0,0 +1,117 @@
+import { Map, Feature, MapBrowserEvent } from 'ol'
+import Style from 'ol/style/Style'
+import Icon from 'ol/style/Icon'
+
+const fallbackHoverFill = '#008A8A'
+const fallbackSelectionFill = '#00A95C'
+
+export const imgSize = [23, 32]
+export const imgSizeMulti = [35, 32]
+
+const prefix = 'data:image/svg+xml,'
+
+/* NOTE path below copied and adapted from
+ * @masterportal/masterportalapi/public/marker.svg */
+
```

Comment:
```suggestion
/**
 * Path of svg copied and adapted from \@masterportal/masterportalapi/public/marker.svg.
 */
```

Reply:
Yeah, that's better. But why the `\`?

Reply:
![no_slash](https://github.com/Dataport/polar/assets/73653210/850f421b-39e5-4875-9ff3-fed2b8052230)

Reply:
Oh, I see. 739ec8a3f9d2012119aeef2635e1ed656414a387 I don't think it should be JSDoc, the other function further below has pretty much the same SVG path, but used three times.